test_launch_ros/test/test_launch_ros/frontend/test_component_container.py

```diff
@@ -0,0 +1,161 @@
+    yaml_file = textwrap.dedent(
+        r"""
+        launch:
+            - node_container:
+                pkg: rclcpp_components
+                exec: component_container
+                name: my_container
+                namespace: ''
+                args: 'test_args'
+                composable_node:
+                    -   pkg: composition
+                        plugin: composition::Talker
+                        name: talker
+                        namespace: test_namespace
+                        remap:
+                            -   from: chatter
+                                to: /remap/chatter
+                        param:
+                            -   name: use_sim_time
+                                value: true
+                        extra_arg:
+                            -   name: use_intra_process_comms
+                                value: 'true'
+
+            - load_composable_node:
+                target: my_container
+                composable_node:
+                    -   pkg: composition
+                        plugin: composition::Listener
+                        name: listener
+                        namespace: test_namespace
+                        remap:
+                            -   from: chatter
+                                to: /remap/chatter
+                        param:
+                            -   name: use_sim_time
+                                value: true
+                        extra_arg:
+                            -   name: use_intra_process_comms
+                                value: 'true'
+        """
+    )
+    with io.StringIO(yaml_file) as f:
+        check_launch_namespace(f)
+
+
+def test_launch_component_container_xml():
+    xml_file = textwrap.dedent(
+        r"""
+        <launch>
+            <node_container pkg="rclcpp_components" exec="component_container" name="my_container" namespace="" args="test_args">
+                <composable_node pkg="composition" plugin="composition::Talker" name="talker" namespace="test_namespace">
+                    <remap from="chatter" to="/remap/chatter" />
+                    <param name="use_sim_time" value="true"/>
+                    <extra_arg name="use_intra_process_comms" value="true"/>
+                </composable_node>
+            </node_container>
+
+            <load_composable_node target="my_container">
+                <composable_node pkg="composition" plugin="composition::Listener" name="listener" namespace="test_namespace">
+                    <remap from="chatter" to="/remap/chatter" />
+                    <param name="use_sim_time" value="true"/>
+                    <extra_arg name="use_intra_process_comms" value="true"/>
+                </composable_node>
+            </load_composable_node>
+        </launch>
+        """  # noqa: E501
+    )
+    with io.StringIO(xml_file) as f:
+        check_launch_namespace(f)
+
+
+def check_launch_namespace(file):
```

Comment:
Maybe the following name is more accurate:

```suggestion
def check_launch_description(file):
```

Reply:
Oh, I forgot to change this after copied from another file!

Reply:
Renamed to `check_launch_component_container` to be consistent with `test_node_frontend.py` and `test_push_ros_namespace_frontend.py`.
440dadc10da4745c784cc2889490cdaf53b46eb1